mirage/config.js

```diff
@@ -464,19 +464,19 @@ export default function() {
   // DELETE /user-categories/:id
   this.delete('/user-categories/:id');
 
-  /////////////
+  // ///////////
   // User roles
-  /////////////
+  // ///////////
 
   // POST /user-roles
   this.post('/user-roles');
 
   // DELETE /user-roles
   this.delete('/user-roles/:id');
 
-  //////////////
+  // ////////////
   // User skills
-  //////////////
+  // ////////////
```

Comment:
This was definitely from using --fix on eslint. I should have given that a closer look. Maybe we should use a different way of making comment blocks stand out, e.g.

```
/**
* User Skills
*/
```
or
```
/*----------------------*
* User Skills
*------------------------/
```
Or we could take out the rule enforce it softly by writing it into a style guide.